<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioRecorder</class>
 <widget class="QMainWindow" name="AudioRecorder">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true"> QPushButton {
         border: 2px solid #8f8f91;
         border-radius: 10px;
         background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                           stop: 0 #f6f7fa, stop: 1 #dadbde);
         min-width: 80px;
		 min-height: 30px;
     }

     QPushButton:pressed {
         background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                           stop: 0 #dadbde, stop: 1 #f6f7fa);
     }

     QPushButton:flat {
         border: none; /* no border for a flat push button */
     }

     QPushButton:default {
         border-color: navy; /* make the default button prominent */
     }


QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}

/******************************/

  QScrollBar:horizontal {
         border: 2px solid grey;
         background: #32CC99;
         height: 15px;
         margin: 0px 20px 0 20px;
     }
     QScrollBar::handle:horizontal {
         background: white;
         min-width: 20px;
     }
     QScrollBar::add-line:horizontal {
         border: 2px solid grey;
         background: #32CC99;
         width: 20px;
         subcontrol-position: right;
         subcontrol-origin: margin;
     }

     QScrollBar::sub-line:horizontal {
         border: 2px solid grey;
         background: #32CC99;
         width: 20px;
         subcontrol-position: left;
         subcontrol-origin: margin;
     }
</string>
  </property>
  <property name="class" stdset="0">
   <string>mainWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">.sss {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.972, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(206, 206, 206, 255))
}</string>
   </property>
   <property name="class" stdset="0">
    <string>sss</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="2">
     <widget class="QPushButton" name="pauseButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="recordButton">
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <layout class="QVBoxLayout" name="levelsLayout"/>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Device:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Current Number: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Current Count: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="nameLE"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="studentIdLE"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="audioDeviceBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="curCntSPB"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="idSPB">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="rangeCB"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Study Group: </string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Student Id:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="curNumSPB"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Given Id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="groupCB"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="outputButton">
      <property name="text">
       <string>Output...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="levelLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Audio Level:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>audioDeviceBox</tabstop>
  <tabstop>groupCB</tabstop>
  <tabstop>rangeCB</tabstop>
  <tabstop>curNumSPB</tabstop>
  <tabstop>curCntSPB</tabstop>
  <tabstop>idSPB</tabstop>
  <tabstop>nameLE</tabstop>
  <tabstop>studentIdLE</tabstop>
  <tabstop>outputButton</tabstop>
  <tabstop>recordButton</tabstop>
  <tabstop>pauseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>outputButton</sender>
   <signal>clicked()</signal>
   <receiver>AudioRecorder</receiver>
   <slot>setOutputLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordButton</sender>
   <signal>clicked()</signal>
   <receiver>AudioRecorder</receiver>
   <slot>toggleRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>AudioRecorder</receiver>
   <slot>togglePause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setOutputLocation()</slot>
  <slot>toggleRecord()</slot>
  <slot>togglePause()</slot>
 </slots>
</ui>
